frontend/tests/controllers/SubmissionsFeedTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ *
+ * @author carlosabcs
+ */
+
+class SubmissionsFeedTest extends \OmegaUp\Test\ControllerTestCase {
+    public function testSubmissionsFeedDAO() {
+        \OmegaUp\Test\Utils::cleanupDB();
+        /**
+         * Create 3 users, 3 problems and 1 contest.
+         *
+         * - User2 is going to have private information, so that user's
+         * submissions should not be returned by the query.
+         *
+         * - Problem2 is going to be private, so their submissions won't
+         * be returned by the query.
+         *
+         * - Contest is going to be unfinished so submissions made over its
+         * problem, won't be returned by the query.
+         */
+
+        $users = [];
+        $identities = [];
+        ['user' => $users[], 'identity' => $identities[]] = \OmegaUp\Test\Factories\User::createUser();
+        ['user' => $users[], 'identity' => $identities[]] = \OmegaUp\Test\Factories\User::createUser(
+            new \OmegaUp\Test\Factories\UserParams([
+            'isPrivate' => true,
```

Comment:
nit:

```suggestion
                'isPrivate' => true,
```

igual en todos los demás lugares donde creas listas (como en L39).